### PR TITLE
update notebooks b/c jax comes with colab now!

### PR DESCRIPTION
**Pull request by mattjj**

### CODE REVIEW

**Review by skye — APPROVED** [review]

**Comment by mattjj** [comment]
Skye you are the BEST, thanks for finding those.

File: notebooks/XLA_in_Python.ipynb [hunk 1]
```diff
@@ -1,966 +1,887 @@
...
-     "data": {
-      "text/plain": [
-       "array(0, dtype=int32)"
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "HMRkxnna8NCN"
+      },
+      "source": [
+        "First install jax and jaxlib to get its xla client:"
```
[the remark this answers is not in the document]
Comment (mattjj) [reply notebooks/XLA_in_Python.ipynb]: Done.

File: notebooks/Common_Gotchas_in_JAX.ipynb [hunk 2]
```diff
@@ -69,36 +69,22 @@
...
     },
     {
+      "cell_type": "markdown",
       "metadata": {
         "id": "n2r1_5_KeiF1",
         "colab_type": "text"
       },
-      "cell_type": "markdown",
       "source": [
         "### Installs and Imports"
```
[the remark this answers is not in the document]
Comment (mattjj) [reply notebooks/Common_Gotchas_in_JAX.ipynb]: Done.

File: notebooks/gufuncs.ipynb [hunk 3]
```diff
@@ -117,20 +117,6 @@
         "### Setup and imports"
```
[the remark this answers is not in the document]
Comment (mattjj) [reply notebooks/gufuncs.ipynb]: Done.

**Pull request merged**